proposed/http-client/http-client-meta.md

```diff
@@ -23,24 +23,23 @@ clients will allow libraries to be decoupled from specific implementation such a
 
 ### Non-Goals
 
-* Support for asynchronous HTTP requests is left for another future PSR;
```

Comment:
Each bullet point is a self-contained sentence, so I ended with a `.` instead of a `;`.